ui/src/app/about/page.tsx

```diff
@@ -0,0 +1,211 @@
+import Image from 'next/image';
+
+const About = () => {
+    return (
```

Comment:
Since nothing is declared above the return statement, we can directly do:
```
const About = () => (
    // ...All the JSX code here
);
```
This is possible because we are using an ES6 arrow function.